libs/ui/src/lib/fire-calculator/fire-calculator.component.ts

```diff
@@ -79,7 +80,7 @@ export class GfFireCalculatorComponent implements OnChanges, OnDestroy {
   @Input() colorScheme: ColorScheme;
   @Input() currency: string;
   @Input() deviceType: string;
-  @Input() fireWealth: number;
+  @Input() fireWealth: FireWealth;
```

Comment:
I would keep this as `number` (it's actually implemented like this in the page, no?)